### PR TITLE
docs:update edge vmware with newer version of palette edge PE-3002

### PR DESCRIPTION
**Pull request by lennessyy**
## Describe the Change

This PR updates the version tags used in the different build commands as well as several configuration documents. It also makes some minor edits to the tutorial. 

## Review Changes

💻 [Deploy on VMWare](https://deploy-preview-1756--docs-spectrocloud.netlify.app/clusters/edge/site-deployment/deploy-cluster)

🎫 [Jira Ticket](https://spectrocloud.atlassian.net/browse/PE-3002)

### CODE REVIEW

File: docs/docs-content/clusters/edge/site-deployment/deploy-cluster.md [hunk 1]
```diff
@@ -80,37 +80,37 @@ You can refer to the [Prepare the DHCP Server for vSphere](https://docs.vmware.c
...
+Clone the [CanvOS](https://github.com/spectrocloud/CanvOS) GitHub repository containing the starter code to build Edge artifacts.
 
 ```bash
 git clone https://github.com/spectrocloud/CanvOS.git
 ```
 
 Change to the **CanvOS** directory.
 
-```bash
+```
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/edge/site-deployment/deploy-cluster.md]:
```suggestion
```shell
```

File: docs/docs-content/clusters/edge/site-deployment/deploy-cluster.md [hunk 2]
```diff
@@ -80,37 +80,37 @@ You can refer to the [Prepare the DHCP Server for vSphere](https://docs.vmware.c
...
 cd CanvOS
 ```
 
-View the available [git tag](https://github.com/spectrocloud/CanvOS/tags).
-<br />
 
-```bash
+View the available [git tags](https://github.com/spectrocloud/CanvOS/tags).
+
+```
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/edge/site-deployment/deploy-cluster.md]:
```suggestion
```shell
```

**Review by karl-cardenas-coding — APPROVED** [review]
Looks good. @lennessyy let's add a sentence or two in the cleanup step to delete the `packer_cache` folder in the user's datastore.

**Comment by lennessyy** [comment]
Looks like there is already instructions to delete packer cache on line 919 - people just have not been following it

**Pull request merged**